.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,46 @@
+          required: true
+    - type: textarea
+      attributes:
+          label: Attachments
+          description: |
+              If possible, please provide any images or videos to help explain the issue you are experiencing. This can include screenshots, screen recordings, or any other relevant files that you think may be helpful.
+      validations:
+          required: false
+    - type: dropdown
+      id: browsers
+      attributes:
+        label: What browsers are you seeing the problem on?
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)